src/index.js

```diff
@@ -0,0 +1,72 @@
+import './style.css';
+import getData from './module/getData.js';
+import { newLike, getLike } from './module/likeApi.js';
+import heartIcon from './images/heart-icon.svg';
+import logo from './images/logo.png';
+
+const availableMovies = document.querySelector('.movie-num');
+const movieList = document.querySelector('.movie-list');
+const logoIcon = document.querySelector('.logo');
+
+logoIcon.src = logo;
+const DisplayLikes = async (item) => {
+  const likesNumber = await getLike();
+  likesNumber.forEach((e) => {
+    if (e.item_id === item.id) {
+      item.querySelector('.num-like').innerHTML = e.likes;
+    }
+  });
+};
+const showMovie = (imageUrl, name, id) => {
+  const item = document.createElement('li');
+  item.classList.add('singleMovie');
+  item.id = id;
+  item.innerHTML = `
+  <img class="show-image" src="${imageUrl}" alt="${name}" />
+  <div class="name-like">
+  <h3 class="movie-name"> ${name} </h3>
+  <img class="like-image" src="${heartIcon}" alt="like" />
+  <p class="num-like"></P>
+  </div>
+  <p class="text-like">Like</P>
+  <div class="btns">
+  <button id="${id}" class="comment-btn btn">Comments</button>
```

Comment:
- [ ] In the comment button on each of the movies' cards, it should display a popup menu with all the comments saved in an API such as the functionality of the likes. Please see the requirement [here](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md#:~:text=the%20given%20mockup.-,Comments%20popup,-When%20the%20popup).